<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScreenRecorder</class>
 <widget class="QWidget" name="ScreenRecorder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="display_area">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>320</width>
     <height>256</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(80,80,80);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListView" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>451</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #0B192C;</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>290</y>
     <width>451</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="connect_btn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>320</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>连接设备</string>
   </property>
  </widget>
  <widget class="QPushButton" name="share_screen">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>360</y>
     <width>321</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>开始投屏</string>
   </property>
  </widget>
  <widget class="QPushButton" name="other_settings">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>400</y>
     <width>321</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>取消投屏</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_fps">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>fps :  </string>
   </property>
  </widget>
  <widget class="QLabel" name="index_fps">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>430</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_1">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>430</y>
     <width>79</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>二值化</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>430</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Server.Version</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>320</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Port:</string>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>display_area</zorder>
  <zorder>line</zorder>
  <zorder>connect_btn</zorder>
  <zorder>share_screen</zorder>
  <zorder>other_settings</zorder>
  <zorder>label_fps</zorder>
  <zorder>index_fps</zorder>
  <zorder>cb_1</zorder>
  <zorder>label</zorder>
  <zorder>lineEdit</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
